backend/api/Services/IsarService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Api.Services
+{
+    public interface IIsarService
+    {
+        public Task<HttpResponseMessage> StartMission(string missionId);
+        public Task<HttpResponseMessage> StopMission(string missionId);
+    }
+    public class IsarService : IIsarService
+    {
+        private readonly string _isarUrl;
```

Comment:
```suggestion
        private readonly string _isarUri;
```
Since the C# way of working with URLs seems to be calling them URIs should we do that too?

Reply:
Yeah, will fix